<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CMainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(128, 128, 128);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(64, 64, 64);</string>
   </property>
   <widget class="QTabWidget" name="mainMenuTabs">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>1261</width>
      <height>761</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="sceneTab">
     <attribute name="title">
      <string>Scene</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>951</width>
        <height>721</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Menlo</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>Scene</string>
      </property>
      <widget class="QWidget" name="m_oglWindow" native="true">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>931</width>
         <height>681</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>23</y>
        <width>271</width>
        <height>697</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Menlo</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(64, 64, 64);</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="landscape_heightmap">
       <attribute name="title">
        <string>Landscape</string>
       </attribute>
       <widget class="QGroupBox" name="generateGroupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>270</y>
          <width>251</width>
          <height>251</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Menlo</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>Generate</string>
        </property>
        <widget class="QSpinBox" name="m_octavesSpinBox">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>100</y>
           <width>81</width>
           <height>24</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>16</number>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>191</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Menlo</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Frequency[0.1 .. 64.0]:</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="m_frequencySpinBox">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>50</y>
           <width>81</width>
           <height>24</height>
          </rect>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>64.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>191</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Menlo</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Octaves[1 .. 16]:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>191</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Menlo</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Seed:</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="m_seedSpinBox">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>150</y>
           <width>81</width>
           <height>24</height>
          </rect>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>65535</number>
         </property>
        </widget>
        <widget class="QPushButton" name="generateButton">
         <property name="geometry">
          <rect>
           <x>23</x>
           <y>200</y>
           <width>201</width>
           <height>32</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Menlo</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="editGroupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>251</width>
          <height>251</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Menlo</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>Edit</string>
        </property>
        <widget class="QSlider" name="m_smoothSlider">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>200</y>
           <width>225</width>
           <height>22</height>
          </rect>
         </property>
         <property name="maximum">
          <number>3</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
        <widget class="QSlider" name="m_brushSizeSlider">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>225</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>4</number>
         </property>
         <property name="maximum">
          <number>32</number>
         </property>
         <property name="singleStep">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>2</number>
         </property>
        </widget>
        <widget class="QLabel" name="m_falloffLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>225</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Menlo</family>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Falloff Coefficient:</string>
         </property>
        </widget>
        <widget class="QLabel" name="m_brushSizeLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>225</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Menlo</family>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Brush Size:</string>
         </property>
        </widget>
        <widget class="QLabel" name="m_smoothLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>180</y>
           <width>225</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Menlo</family>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Smooth Coefficient:</string>
         </property>
        </widget>
        <widget class="QSlider" name="m_brushStrengthSlider">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>225</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
        <widget class="QLabel" name="m_brushStrengthLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>225</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Menlo</family>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Brush Strength:</string>
         </property>
        </widget>
        <widget class="QSlider" name="m_falloffSlider">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>225</width>
           <height>22</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>10</number>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="landscape_textures">
       <attribute name="title">
        <string>Textures</string>
       </attribute>
       <widget class="QPushButton" name="m_texture01Btn">
        <property name="geometry">
         <rect>
          <x>142</x>
          <y>40</y>
          <width>91</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Open...</string>
        </property>
       </widget>
       <widget class="QLabel" name="m_texture01Label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>221</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Menlo</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Texture sampler 1:</string>
        </property>
       </widget>
       <widget class="QLabel" name="m_texture02Label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>221</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Menlo</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Texture sampler 2:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="m_texture02Btn">
        <property name="geometry">
         <rect>
          <x>142</x>
          <y>210</y>
          <width>91</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Open...</string>
        </property>
       </widget>
       <widget class="QLabel" name="m_texture03Label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>350</y>
          <width>221</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Menlo</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Texture sampler 3:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="m_texture03Btn">
        <property name="geometry">
         <rect>
          <x>142</x>
          <y>380</y>
          <width>91</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Open...</string>
        </property>
       </widget>
       <widget class="QLabel" name="m_texture01Img">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>128</width>
          <height>128</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="m_texture02Img">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>128</width>
          <height>128</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="m_texture03Img">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>380</y>
          <width>128</width>
          <height>128</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QSpinBox" name="m_textureTilling01SpinBox">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>100</y>
          <width>91</width>
          <height>24</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>128</number>
        </property>
       </widget>
       <widget class="QLabel" name="m_textureTilling01Label">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>80</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Menlo</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Tilling:</string>
        </property>
       </widget>
       <widget class="QLabel" name="m_textureTilling02Label">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>250</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Menlo</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Tilling:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="m_textureTilling02SpinBox">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>270</y>
          <width>91</width>
          <height>24</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>128</number>
        </property>
       </widget>
       <widget class="QLabel" name="m_textureTilling03Label">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>420</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Menlo</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Tilling:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="m_textureTilling03SpinBox">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>440</y>
          <width>91</width>
          <height>24</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>128</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="models">
       <attribute name="title">
        <string>Models</string>
       </attribute>
       <widget class="QWidget" name="m_modelsOpenGLView" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>256</width>
          <height>256</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="materialsTab">
     <attribute name="title">
      <string>Materials</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
